assets/js/modules/collection.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
+
+import Book from './book.js';
+
+const titleId = document.getElementById('title');
+const authorId = document.getElementById('author');
+const bookContainer = document.getElementById('book_container');
+const formBook = document.getElementById('book_form');
+
+export default class Collection {
+  constructor() {
+    this.library = [];
+  }
+
+  addBook = () => {
+    const id = Math.floor(Math.random() * 10000);
+    const newBook = new Book(id, titleId.value, authorId.value);
+    this.library.push(newBook);
+    this.addStorage(this.library);
+    displayBook(this.library);
+  }
+
+  /* eslint-disable class-methods-use-this */
```

Comment:
- Please fix all disabled linter checks, remember linters help us to ensure that our code is clean and easy to maintain. In this instance you should name your `addStorage` method as `static addStorage` since the method does not make use of the `this` keyword in its code block. Please fix this 👍 👍 .
## Note: when calling a method that uses the `static` keyword, you call the method with the name of the class, i.e `className.methodName`